app/models/manageiq/providers/openshift/container_manager/refresher.rb

```diff
@@ -3,9 +3,18 @@ module Openshift
     class ContainerManager::Refresher < ManageIQ::Providers::BaseManager::Refresher
       include ::EmsRefresh::Refreshers::EmsRefresherMixin
 
+      KUBERNETES_EMS_TYPE = ManageIQ::Providers::Kubernetes::ContainerManager.ems_type
```

Comment:
@moolitayer @simon3z I would ask that we remove this constant until we actually need it.